client/src/routes/login.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+	import Textfield, { HelperLine } from '@smui/textfield';
+	import Button, { Label as ButtonLabel } from '@smui/button';
+	import Snackbar, { Label as SnackbarLabel, SnackbarComponentDev } from '@smui/snackbar';
+	import { form, field } from 'svelte-forms';
+	import { required } from 'svelte-forms/validators';
+	import { user_api } from '../api/user';
+
+	let failureSnackbar: SnackbarComponentDev;
+
+	const user = field('user', '', [required()]);
+	const password = field('password', '', [required()]);
+	const loginForm = form(user, password);
+
+	function login() {
+		if ($loginForm.valid && $user.value && $password.value) {
+			user_api
+				.login({ user: $user.value, password: $password.value })
+				.then((res) => {
+					// TODO something
```

Comment:
Good question. I think we need to save the cookie manually, which is in the header.
Although, its better to do that in the mock, as Axios should take care of that with the real responses.

Probably nothing to do here.

Reply:
Then we will cross that cookie when we have get there. Also the endpoint thingy should eliminate this part of the code completely, but I could not figure it out yet.